junebug/channel.py

```diff
@@ -193,9 +193,17 @@ def _transport_config(self):
 
     @property
     def _application_config(self):
+        redis_manager = self.redis._config
+        # For tests, FAKE_REDIS cannot be an object, else deepcopy doesn't
+        # work.
+        if 'FAKE_REDIS' in redis_manager:
+            redis_manager['FAKE_REDIS'] = 'yes'
```

Comment:
Ah, I see. Hmm, wondering if it would okay to just use `self.config.redis` here for `redis_manager`. I might be missing a reason why that wouldn't work though.